benchmarks/src/main/kotlin/bfschannel/BfsChannelBenchmark.kt

```diff
@@ -0,0 +1,363 @@
+@file:JvmName("BfsChannelBenchmark")
+
+package bfschannel
+
+
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.channels.*
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import org.apache.commons.math3.stat.descriptive.moment.StandardDeviation
```

Comment:
This library is better in our case 
https://github.com/thomasnield/kotlin-statistics